### PR TITLE
fix graph stage logging docs

### PR DESCRIPTION
**Pull request by ortex**
fix graph stage logging docs

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @ortex,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by ktoso** [comment]
OK TO TEST

**Review by ktoso — APPROVED** [review]
Thanks!

File: akka-docs/src/main/paradox/stream/stream-customize.md [hunk 1]
```diff
@@ -301,8 +301,8 @@ In that sense, it serves a very similar purpose as `ActorLogging` does for Actor
 
 @@@ div { .group-java }
 
-You can extend the `akka.stream.stage.GraphStageWithLogging` or `akka.strea.stage.TimerGraphStageWithLogging` classes
```
Comment (raboof) [inline akka-docs/src/main/paradox/stream/stream-customize.md]: Good improvement. I think we can even do one better: by changing these to `@unidoc[GraphStageWithLogging]` they will be linked to the javadoc, and checked for typo's each time the docs are generated. Could you give that a try?

Comment (ortex) [reply akka-docs/src/main/paradox/stream/stream-customize.md]: Yep.
I added `@unidoc` for `GraphStageWithLogging` and `TimerGraphStageWithLogging `

**Review by raboof — APPROVED** [review]
Excellent improvement, thanks!

**Pull request merged**